docs/add-ins/get-and-set-recurrence.md

```diff
@@ -0,0 +1,190 @@
+---
+title: Get and set recurrence in an Outlook add-in
```

Comment:
This PR should also include an update to the TOC file, to add this new article to the TOC.

Reply:
Will do.